front-end/projet_application/src/routes/resetPassword/+page.svelte

```diff
@@ -3,27 +3,60 @@
   import "../../styles/resetPassword.css";
   import "../../styles/global.css";
   import type { ResetPassword } from "../../Models/ResetPassword.ts";
-  let error = "";
+  import type { ErrorResponse } from "../../Models/ErrorResponse.ts";
+  import * as yup from "yup";
+
+  const schema = yup.object().shape({
+    password: yup.string().required("Entrer un mot de passe"),
+    confirmPassword: yup.string().required("Saisir le mot de passe à nouveau"),
```

Comment:
On devrait y mettre une validation pour que le mot de passe soit le même et aussi une REGEX pour le mot de passe. 
Pour la validation du même mot de passe : https://stackoverflow.com/questions/61862252/yup-schema-validation-password-and-confirmpassword-doesnt-work

Reply:
Les deux sont faits!